puppet/zulip/files/postgresql/process_fts_updates

```diff
@@ -43,6 +43,8 @@ sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../.
 try:
     os.environ['DJANGO_SETTINGS_MODULE'] = 'zproject.settings'
     from django.conf import settings
+    if settings.USING_PGROONGA:
+        exit()
```

Comment:
OK. I've implemented and pushed it.

In the implementation, the default full-text search index is also updated. Should we disable index update for the default full-text search?